space.nouspiro.tenmon.json

```diff
@@ -15,7 +15,7 @@
 		"--share=ipc",
 		"--socket=x11",
 		"--device=dri",
-		"--filesystem=host:ro",
+		"--filesystem=host:rw",
 		"--filesystem=xdg-run/gvfs:ro"
```

Comment:
Application also enable to filter and organize files by moving and coping so it need also write files